src/content/posts/elastic-ilm-test.md

```
@@ -0,0 +1,291 @@
+
+- ILM is applied on indices using ILM policies.
+- ILM policies contain information that takes action on data based on the threshold guidelines mentioned in the policy.
+- ILM policies contain phases (hot, warm, cold, delete). You can perform specific actions along with the data transition, like reducing the number of primary shards and taking a snapshot.
+
+### Test driving ILM
+
+[Elastic.co](http://elastic.co) has detailed ILM [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html). In this blog post, we'll see how ILM works. I want to show how data could be transitioned from one phase (tier: hot) to another (tier: warm). 
+
+#### Prerequisites: 
+
+- Create a free Elastic Cloud account [here](https://cloud.elastic.co/registration).  Spin up an Observability deployment with a warm tier → launch Kibana → Head to "Dev Tools" in the left menu. Refer to screenshots [here](#elastic-cloud-account-creation  ).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)